src/getProcedureDump.ts

```diff
@@ -0,0 +1,81 @@
+import { ProcedureDumpOptions } from './interfaces/Options';
+import { DB } from './DB';
+
+interface ShowProcedures {
+    Name: string;
+    sql_mode: string;
+    definer: string;
+    character_set_client: string;
+    coallation_connection: string;
+    'Database Collation': string;
+}
+interface ShowCreateProcedure {
+    Procedure: string;
+    sql_mode: string;
+    'Create Procedure': string;
+    character_set_client: string;
+    coallation_connection: string;
```

Comment:
Wrong definition. It should be ```collation_connection```
```suggestion
    collation_connection: string;
```